jiant/__main__.py

```diff
@@ -567,7 +567,20 @@ def main(cl_arguments):
     if args.do_pretrain:
         # Train on pretrain tasks
         log.info("Training...")
-        stop_metric = pretrain_tasks[0].val_metric if len(pretrain_tasks) == 1 else "macro_avg"
+        if args.early_stopping_method != "auto":
```

Comment:
this is the right direction, but you may want move it somewhere else, otherwise 
```
elif args.early_stopping_method in target_names:
                index = target_names.index(args.early_stopping_method)
                stop_metric = target_tasks[index].val_metric
```
won't have any effect.

Reply:
What do you mean? This should only be in pretrain_tasks because multitask learning is only in pretraining.

Reply:
In that case, why will there be a case for early_stopping_method in target_names but not in pretrain_names?

Reply:
Ah yeah, there wouldn't. I'll take it out